about/initiative.md

```diff
@@ -40,16 +40,17 @@ We understand that integrating new technologies can be challenging. That's why O
 Getting started with Open Gateway APIs is easy:
 
 1. **Explore Available APIs:** Browse our API catalog to discover the various network capabilities and services that you can integrate into your applications.
-2. **Find a channel partner:** Identify a Channel Partner or join the [Developer Hub](https://opengateway.telefonica.com/developer-hub) in order to use the APIs through its Sandbox environment.
+2. **Find a channel partner or use our Sandbox:** Identify a channel partner or join to the [Developer Hub](https://opengateway.telefonica.com/developer-hub) in order to use the APIs. 
```

Comment:
We'd better always capitalize the term `Channel Partner` as a specific GSMA term.

I think `join` is correct over `join to`